src/Domain/RegularGivingNotifier.php

```diff
@@ -36,22 +40,58 @@ public function notifyNewMandateCreated(
                     'amount' => $mandate->getAmount()->format(),
-                    'giftAidValue' => '',   // @todo-regular-giving: think about where best to calculate GA value.
-                                            // if government changes tax code after mandate started.
-                    'totalIncGiftAd' => '',
+            //                    'giftAidValue' => '',   // @todo-regular-giving: think about where best to calculate GA value.
+            //                                            // if government changes tax code after mandate started.
+            //                    'totalIncGiftAd' => '',
                     'totalCharged' => $mandate->getAmount()->format(),
-                    'charityPostalAddress' => '',
-                    'charityPhoneNumber' => '',
-                    'charityEMailAddress' => '',
-                    'charityWebsite' => '',
-
-                    'firstDonation' => [
-                        // @todo-regular-giving: Fill in details of first donation from mandate, which should have
-                        // been collected before we send this email. We will be forced to violate DRY since the existing
-                        // equivalent for sending donation notifications is in SF not matchbot.
-                    ]
+
+                    'firstDonation' => $this->donationToConfirmationEmailFields(
+                        $firstDonation,
+                        $charity,
+                        $campaign
+                    )
                 ],
             ]
         );
     }
+
+    /**
+     * This is currently of course just for regular giving, but we may start using matchbot to send
+     * ad-hoc donation email reciepts in future, in which case this function should be able to be moved
+     * and re-used
+     */
+    private function donationToConfirmationEmailFields(
+        Donation $firstDonation,
+        Charity $charity,
+        Campaign $campaign
+    ): array {
+        $firstDonationCollectedAt = $firstDonation->getCollectedAt();
+
+        // @todo-regular-giving add assertion:
+        // Assertion::notNull($firstDonationCollectedAt);
+        // @see \MatchBot\Domain\RegularGivingService::setupNewMandate
+
+        return [
+            'currencyCode' => $firstDonation->getCurrencyCode(),
+            'donationAmount' => $firstDonation->getAmount(),
+            'donationDatetime' => $firstDonationCollectedAt?->format('c'),
+            'charityName' => $charity->getName(),
+            'transactionId' => $firstDonation->getTransactionId(),
+            'matchedAmount' => $firstDonation->getFundingWithdrawalTotal(),
+            'statementReference' => $campaign->getCharity()->getStatementDescriptor(),
+
+            // @todo-regular-giving - implement giftAidAmountClaimed. I think we want to store
+            // this in the DB, not rely on being able to calculate in demand, to allow for the
+            // tax rate to change.
```

Comment:
giftAidAmountClaimed is used in mailer's existing template for non-regular giving emails here:

https://github.com/thebiggive/mailer/blob/75ce26214ad719f794b44c29cf4b910805c6bdd5/templates/donor-donation-success.html.twig#L37 . 

I think the past tense 'claimed' is maybe a misnomer, or it means claimed by the donor rather than claimed already from HMRC.